src/ts/components/subtitleoverlay.ts

```diff
@@ -318,9 +320,22 @@ class ActiveSubtitleManager {
   cueEnter(event: SubtitleCueEvent): SubtitleLabel {
     let id = ActiveSubtitleManager.calculateId(event);
 
+    let generateImageTagText = (imageData: string): string => {
```

Comment:
Move this to a `private static` function to avoid recreating this function on every `cueEnter`.